CONTRIBUTING.md

```diff
@@ -1,227 +1,150 @@
 # Contribute to Strapi
 
-> ⚠️ We have stopped merging PRs for now to the Strapi core.<br><br>
-> The reason is that we are developing new architecture for the admin panel and for the plugins.<br>
-> This new architecture will provide stability of the Strapi core as we approach the release of Beta.<br>
-> We appreciate and welcome all your contributions, but until further notice, please do not submit a PR as it will not be merged.<br>
-> Furthermore, you will have to rewrite it based on the new architecture.
-
 First off, thanks for taking the time to contribute! 🎉👍
 
 The following is a set of guidelines for contributing to Strapi and its packages.
 
 Strapi is an open-source project administered by [the Strapi team](https://strapi.io/company).
 
-Before contributing, ensure that your effort is aligned with the project's roadmap by talking to the maintainers, especially if you are going to spend a lot of time on it. Feel free to [join us on Slack](http://slack.strapi.io) if you are interested in helping us or [drop us an email](mailto:hi@strapi.io) if you are interested in working with us.
-
-## Code of Conduct
-
-This project and everyone participating in it is governed by the [Strapi Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [hi@strapi.io](mailto:hi@strapi.io).
+Before contributing, ensure that your effort is aligned with the project's roadmap by talking to the maintainers, especially if you are going to spend a lot of time on it. Feel free to [join us on Slack](http://slack.strapi.io) or [drop us an email](mailto:hi@strapi.io) if you are interested in helping us or working with us.
 
 ## Open Development & Community Driven
+
 Strapi is open-source under the [MIT license](https://github.com/strapi/strapi/blob/master/LICENSE.md). All the work done is available on GitHub.
 The core team and the contributors send pull requests which go through the same validation process.
 
 Every user can send a feature request using the [issues](https://github.com/strapi/strapi/issues/new?template=FEATURE_REQUEST.md) on GitHub. Feel free to upvote 👍 [existing feature request](https://portal.productboard.com/strapi)
```

Comment:
Feel free to upvote 👍[existing feature request**s**](https://portal.productboard.com/strapi)**.**

Two changes an "s" and a "."